src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheckTest.java

```diff
@@ -76,7 +79,8 @@ public void testCheck() throws Exception {
             "322: " + getCheckMessage(MSG_KEY, 4),
             "324: " + getCheckMessage(MSG_KEY, 4),
         };
-        verify(checkConfig, getPath("InputJavaDocTagContinuationIndentation.java"),
+        verify(checkConfig,
+                getPath("InputJavaDocTagContinuationIndentation.java"),
```

Comment:
File name didn't change so this line change is completely unnecessary.
Please undo change.